showkase/src/main/java/com/airbnb/android/showkase/ui/ShowkaseCategoriesScreen.kt

```diff
@@ -82,6 +83,9 @@ internal fun goBackToCategoriesScreen(
         showkaseBrowserScreenMetadata.value.isSearchActive -> {
             showkaseBrowserScreenMetadata.clearActiveSearch()
         }
+        navController.currentDestination?.id == navController.graph.startDestinationId -> {
+            activity.finish()
```

Comment:
I didn't follow this use case. Could you elaborate on which scenario we are trying to close the activity for?

Reply:
This happen because we go directly to the Component screen, so we are already on the root of the nav graph, so when we click back, we don't have to navigate to SHOWKASE_CATEGORIES, but finish.